common/handler_cloud.py

```diff
@@ -46,7 +46,9 @@ def get_best_score_imgs(self, results: list[ScoredPoint]) -> list[Image.Image]:
         Handler for returning images with the highest similarity scores from cloud storage.
         Additionally, filenames are returned as future captions in front-end module.
         """
-        object_list = [MINIO_MAIN_PATH / r.payload["file"].replace('\\', '/') for r in results]
+        object_list = [
+            MINIO_MAIN_PATH / r.payload["file"].replace("\\", "/") for r in results
```

Comment:
Why are you doing replace here? Isn't it better to use pathlib and just cast to posix path?

Reply:
`r.payload` comes from Qdrant and if you generate entries on Windows, current indexing will pull the Windows path with `\` as a folder separator. This should be left as a precaution when working locally on Windows.

Reply:
I'll resolve it on the next major data update.